.travis.yml

```diff
@@ -4,9 +4,14 @@ cache:
     - node_modules
 script:
   - script/test
+before_deploy:
+  - "export NPM_PACKAGE_VERSION=$(semver $(cat package.json | jsawk 'return this.version')"
+  - "export GIT_PACKAGE_VERSION=$(semver $(git describe --tags))"
```

Comment:
Does this git command work for you? I'd expected something like `git rev-parse --short HEAD`.